devstack/lib/nfp

```diff
@@ -196,8 +196,8 @@ function create_nfp_gbp_resources {
 
     gbp l3policy-create\
  --ip-version 4\
- --proxy-ip-pool=192.169.0.0/24\
- --ip-pool 120.0.0.0/24\
+ --proxy-ip-pool=192.168.0.0/16\
```

Comment:
Remove --proxy-ip-pool here. It's ok to pick the default (defined in ./neutron/extensions/driver_proxy_group.py)